Code/Tools/SceneAPI/SceneUI/SceneWidgets/ManifestWidgetPage.cpp

```diff
@@ -469,6 +469,35 @@ namespace AZ
                 }
             }
 
+            void ManifestWidgetPage::AddObjects(AZStd::vector<AZStd::shared_ptr<DataTypes::IManifestObject>>& objects)
+            {
+                ManifestWidget* parent = ManifestWidget::FindRoot(this);
+                AZ_Assert(parent, "ManifestWidgetPage isn't docked in a ManifestWidget.");
+                if (!parent)
+                {
+                    return;
+                }
+                AZStd::shared_ptr<Containers::Scene> scene = parent->GetScene();
+                if (!scene)
+                {
+                    return;
+                }
+                Containers::SceneManifest& manifest = scene->GetManifest();
+                for (auto& object : objects)
+                {
+                    if (!SupportsType(object))
+                    {
+                        continue;
+                    }
+                    if (!manifest.AddEntry(object))
+                    {
+                        AZ_Assert(false, "Unable to add new object to manifest.");
+                    }
```

Comment:
If AddObjects can be executed from a Python script, then this AZ_Assert() will crash the Editor based on a script error, right? Is that the behavior we want here?

Reply:
It won't crash, but the whole tool will be in an unstable state with information not in the manifest that is in the widget page.

I'll downgrade it to an error, but if this happens, then things are broken and you'll want to re-start your Editor or find an engineer to debug it.